### PR TITLE
Fix DropDownEditor Popup position depending the "rtlEnabled" option value (T856114)

### PR DESCRIPTION
**Pull request by DokaRus**

### CODE REVIEW

File: testing/tests/DevExpress.core/utils.position.tests.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import config from 'core/config';
```
Comment (San4es) [inline testing/tests/DevExpress.core/utils.position.tests.js:1]: Let's merge this file with this one:
https://github.com/DevExpress/DevExtreme/blob/fefd39b7ef1acf829100378d8aec47ea384f1bce/testing/tests/DevExpress.animation/position.tests.js

Comment (DokaRus) [reply testing/tests/DevExpress.core/utils.position.tests.js:1]: I don't think we should do that. The positioning utilities located in `animation/utils` are used to support the animation module.
This one (`core/position`) is private utility, used to get start direction or [base alignment](https://github.com/DevExpress/DevExtreme/blob/20_1/js/ui/data_grid/ui.data_grid.export.js#L113) based on passed argument or global `rtlEnabled` config.

Comment (San4es) [reply testing/tests/DevExpress.core/utils.position.tests.js:1]: Thanks! My mistake...

**Review by San4es — APPROVED** [review]
LGTM

**Review by alexander-kotov-dx — APPROVED** [review]

**Pull request merged**